i18n-js.gemspec

```diff
@@ -18,9 +18,7 @@ Gem::Specification.new do |s|
   s.executables   = `git ls-files -- bin/*`.split("\n").map{ |f| File.basename(f) }
   s.require_paths = ["lib"]
 
-  # CVE-2014-10077 which is fixed for >= 0.8.0
-  # https://nvd.nist.gov/vuln/detail/CVE-2014-10077
-  s.add_dependency "i18n", ">= 0.8.0", "< 2"
+  s.add_dependency "i18n"
```

Comment:
This does not just rollback the i18n requirement change
Before https://github.com/fnando/i18n-js/commit/b0d917acf605ba5c1615bde38ebbe1ff1bb3f12b#diff-03ae15123ab415633ccea502d7131c4a it was requiring `>= 0.6.6`